### PR TITLE
remove_deprecated_part_2

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi @tinatian , @shuyangzhou ,

I have removed deprecated methods or classes for "**portal-impl/src/com/liferay/portal/dao**",  "**portal-impl/src/com/liferay/portal/security**", "**portal-impl/src/com/liferay/portal/social**", and "**portal-impl/src/com/liferay/portal/model/impl**".
I also removed "portal-impl/src/com/liferay/portal/security/jaas/ext/BasicLoginModule" in source-formatter-suppressions.xml since it is deleted.
see https://github.com/jeffwu0724/liferay-portal/pull/20.

Thanks,
Jeff

### CODE REVIEW

**Comment by jeffwu0724** [comment]
ci:test:sf

**Comment by jeffwu0724** [comment]
ci:test:relevant

**Comment by jeffwu0724** [comment]
ci:test

**Comment by tinatian** [comment]
@jeffwu0724 Code conflicts, please rebase.

Thanks.
Tina

**Comment by jeffwu0724** [comment]
@kyle-miho Hey Kyle, could you please help me to scan the test result?

Thanks,
Jeff

**Comment by kyle-miho** [comment]
@jeffwu0724 this pr looks good to go

**Comment by jeffwu0724** [comment]
> @jeffwu0724 this pr looks good to go

Thank you Kyle!

File: portal-kernel/src/com/liferay/portal/kernel/model/Company.java [hunk 1]
```diff
@@ -101,12 +101,6 @@ public String getShortName()
 
 	public boolean isAutoLogin();
 
-	/**
```
Comment (tinatian) [inline portal-kernel/src/com/liferay/portal/kernel/model/Company.java:104]: @jeffwu0724 Company and CompanyWrapper are generated classes, usually we did not modify them manually. I will accept the code this time as it is a small change. Next time, please modify CompanyImpl, remove the method there, then run build-service to generate the change on Company and CompanyWrapper

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina.

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/113577

**Comment by shuyangzhou** [comment]
ci:close

**Pull request closed without merging**